### DISCUSSION TITLE
[Weekly Meeting] [2024-03-01] - Meeting Notes

### DISCUSSION THREAD

**Discussion by porcelli** [apache/incubator-kie-issues#971] [Weekly Meetings]
Proposed Topics:

- Community
  - [ ] SonataFlow Logo - designer working on it. Ricardo provided some input, may have something next week! yay!
- Infrastructure
   - [ ] code signing certificate [Pending on Alex] - https://issues.apache.org/jira/browse/INFRA-25549
- [Category X Dependencies not allowed](https://www.apache.org/legal/resolved.html#category-x)  
  - [ ] [Hibernate ORM!!!!](https://hibernate.org/orm/)
     - [ ] Drools 
          - [ ] https://github.com/apache/incubator-kie-drools/blob/main/build-parent/pom.xml#L780-L795
          - [ ] https://github.com/apache/incubator-kie-drools/blob/main/build-parent/pom.xml#L1102-L1105
     - [ ] Apps 
          - [ ] https://github.com/apache/incubator-kie-kogito-apps/blob/main/kogito-apps-build-parent/pom.xml#L242-L246
     - [ ] AI: Reach out Hibernate team progress on ASLv2
     - [ ] AI: Reach Apache IPMC if we can have initial releases with Hibernate
     - [ ] AI: We eventually need adjust code to remove hibernate
- Quarkus 
   - [ ] 3.8.0 investigation
   - [ ] 3 known issues: Operator, Hibernate more strict, DevUI new API.
   - [ ] Operator integration test. Ricardo has experience, but with Go. Java seems harder.
- Test failures
   - [ ] Native failings
   - [ ] Flaky tests
- Rebranded Quarkus Extensions [10-blocker] 
   - [ ] Massive change with Kogito and KIE changes
   - [ ] Temp Revert SonataFlow - in discussion with Apache INFRA - https://issues.apache.org/jira/browse/INFRA-25546
- Reproducible builds [10-blocker]
   - [ ] KIE Tools Circular Dependencies
       - [ ] Very positive: Green build with Java 17, things looking good to be checked manually
       - [ ] Move examples that have dependencies to devui to kie-tools repo.
       - [ ] Trusty still unknown - probably remove and bring back when we have capacity to work on it.
- Mario quick Recap: Hibernate investigate, Circular dependency remaining blockers (Hibernate might not be, need check).

----

Recording

https://us06web.zoom.us/rec/share/_MvUv8twJUAh7MjRPG6K2nrrdmLea3D6F_omhcRLm6WMX0Yv7zOQrTYgc0k-BhO3.JNDD5jY9pQEoewZ-
Passcode: T57b9ZX.

**Comment by LightGuard** [comment]
I asked on Hibernate's Zulip about their progress on ASL 2.0: https://hibernate.zulipchat.com/#narrow/stream/132096-hibernate-user/topic/ASL.202.2E0.3F

**Reply by LightGuard** [reply]
I don't know what all we're using Hibernate for, but maybe it's possible to use Apache's JPA impl for the APIs?
